domain-server/src/DomainServer.cpp

```diff
@@ -1506,11 +1505,24 @@ QJsonObject jsonForDomainSocketUpdate(const HifiSockAddr& socket) {
 
 const QString DOMAIN_UPDATE_AUTOMATIC_NETWORKING_KEY = "automatic_networking";
 
-void DomainServer::performIPAddressUpdate(const HifiSockAddr& newPublicSockAddr) {
-    sendHeartbeatToMetaverse(newPublicSockAddr.getAddress().toString());
+void DomainServer::performIPAddressPortUpdate(const HifiSockAddr& newPublicSockAddr) {
+    static const QString PUBLIC_SOCKET_ADDRESS_KEY = "network_address";
+    static const QString PUBLIC_SOCKET_PORT_KEY = "network_port";
```

Comment:
This should be moved out to the top of the file, since it's duplicated below, and that goes against the point of having a constant.